src/core/Memory.cc

```diff
@@ -35,7 +31,7 @@ Memory::~Memory()
     // needed to release memory (and can't be passed in here).  So to
     // release the memory, an explicit clear must called using the
     // queue parameter ( Memory::clearDeviceBlocks(device, *queue) ).
-    assert(capacity_[ HostNum ] == 0);
+    //assert(capacity_[ HostNum ] == 0);
```

Comment:
Since commented, do we want to remove it?

Reply:
All of the host memory pool code is commented out instead of being removed,, so I was following that convention.  There's a broader discussion that needs to happen about if SLATE should use a memory pool for host data or if that code should properly be removed.